util/cron/common-arkouda.bash

```diff
@@ -48,7 +48,7 @@ export CHPL_WHICH_RELEASE_FOR_ARKOUDA="1.30.0"
 # test against Chapel release (checking out current test/cron directories)
 function test_release() {
   export CHPL_TEST_PERF_DESCRIPTION=release
-  export CHPL_TEST_PERF_CONFIGS="release:v,nightly"
```

Comment:
This makes "release" not visible by default. See `--configs` option in `./util/test/genGraphs --help`

Reply:
Were you thinking that we would have both of them visible by default? I was thinking we only wanted to look at nightly by default.

Reply:
I was thinking both, but let's discuss in perf meeting today

Reply:
Switched to that, that's fine with me.